pkgs/native_assets_builder/test/helpers.dart

```diff
@@ -154,13 +154,17 @@ Future<void> copyTestProjects({
     final sourceString = await sourceFile.readAsString();
     final modifiedString = sourceString.replaceAll(
       'path: ../../',
-      'path: ${pkgNativeAssetsBuilderUri.toFilePath().replaceAll('\\', '/')}',
+      'path: ${pkgNativeAssetsBuilderUri.toFilePath().unescape()}',
     );
     await File.fromUri(targetFileUri)
         .writeAsString(modifiedString, flush: true);
   }
 }
 
+extension UnescapePath on String {
+  String unescape() => replaceAll('\\', '/');
```

Comment:
Please add a doc comment on what unescape does and whether it works for JSON/YAML/... ? Windows/non-Windows?